.github/DISCUSSION_TEMPLATE/help.yml

```diff
@@ -1,25 +1,36 @@
----
 body:
-  - type: textarea
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to start a new discussion!
+        
+        Before you post:
+        • Check if someone has already asked/answered your question in a previous discussion.
+        
+        When you're ready to post:
+        • Add tags to your discussion (e.g. 'React', 'Vue', 'Vite') to make it clearer for other users.
+        
+  - type: textarea 
     attributes:
       label: Summary
-      description: How do you need help?
-      placeholder: I need help with...
-      validations:
+      description: How can we help?
+    validations:
         required: true
+        
   - type: textarea
     attributes:
       label: Additional information
-      description: Share your Storybook configuration (`main.js or ts`), any error messages, or relevant dependencies
+      description: |
+        Share Your Storybook configuration (`main.js or main.ts`), your Storybook version number, any error messages, or any relevant dependencies. These help us get a clearer understanding of what might be going wrong.
+
+        P.s. please share code as text rather than as a screenshot! It makes debugging much easier and faster.
```

Comment:
```suggestion
        P.s. please [share code as text](https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/creating-and-highlighting-code-blocks) rather than as a screenshot! It makes debugging much easier and faster.
```